src/index.ts

```diff
@@ -518,6 +518,8 @@ export interface GlobalAuroraRDSMasterProps {
 export interface RegionalOptions {
   readonly region: string;
   readonly dbSubnetGroupName?: string;
+  readonly dbClusterpPG?: string;
```

Comment:
```suggestion
  readonly dbParameterGroup?: string;
```

Reply:
Updated, as you already had that variable name for cluster parameter group in Master stack in same file, I just copied it over for consistency, but I agree using the name I added in the Python `Custom Resource` is more appropriate.